merge in master

junghans opened the pull request:
Just to avoid that they diverge more.

12AngryMen commented:
Hallo Christoph,

kannst du mir mal fuer dumme erklaeren was da mit dem merge passiert ist.
Danke

LG Jens

2015-10-02 0:59 GMT+02:00 Christoph Junghans <notifications@github.com>:

> Just to avoid that they diverge more.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/votca/ctp/pull/77
> Commit Summary
>
>    - Closing old branch
>    - Closing old branch
>    - trying to merge
>    - merged with carl
>    - Fixed 'trimming' to larger number of unoccupied states than available
>    - merge two default heads from Bjoern & Pascal
>    - dummy merge with Denis default branch
>    - dummy merge with Denis even older default branch
>    - cmake: updated modules
>    - cmake: allow version from git, too
>    - remove auto-generated Doxyfile
>    - fixed some format warnings
>    - ctp_testsuite, ctp_update only support python2
>    - fixed many many warnings
>    - gcc warning: added LOG_SAVE
>    - fixed some more warning found by travis
>    - fixed even some more warning found by travis
>    - some more warning found by travis
>    - fixed more warning
>    - fixed a bug introduced in b9ba31880eb5
>    - merge with googlecode
>    - fixed more bugs introduced in b9ba31880eb5
>    - ctp_dump rates, minor bug fix
>    - updated CHANGELOG.md
>    - merge with googlecode
>    - added .travis.yml
>    - hg -> git
>    - cmake: some tweaks for global build
>    - cmake: fixed an out-of-source issue
>    - .travis.yml: update
>    - Merge branch 'master' into wehnerj
>
> File Changes
>
>    - *R* .gitignore <https://github.com/votca/ctp/pull/77/files#diff-0>
>    (5)
>    - *D* .hgtags <https://github.com/votca/ctp/pull/77/files#diff-1> (1)
>    - *A* .travis.yml <https://github.com/votca/ctp/pull/77/files#diff-2>
>    (61)
>    - *R* CHANGELOG.md <https://github.com/votca/ctp/pull/77/files#diff-3>
>    (16)
>    - *M* CMakeLists.txt
>    <https://github.com/votca/ctp/pull/77/files#diff-4> (18)
>    - *D* CMakeModules/FindVOTCA_BOOST.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-5> (37)
>    - *M* CMakeModules/FindVOTCA_CSG.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-6> (22)
>    - *D* CMakeModules/FindVOTCA_KMCOLD.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-7> (59)
>    - *M* CMakeModules/FindVOTCA_MOO.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-8> (6)
>    - *M* CMakeModules/FindVOTCA_TOOLS.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-9> (22)
>    - *M* CMakeModules/hgscript.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-10> (25)
>    - *M* CMakeModules/hgversion.cmake
>    <https://github.com/votca/ctp/pull/77/files#diff-11> (26)
>    - *M* include/votca/ctp/apolarsite.h
>    <https://github.com/votca/ctp/pull/77/files#diff-12> (11)
>    - *M* include/votca/ctp/atom.h
>    <https://github.com/votca/ctp/pull/77/files#diff-13> (37)
>    - *M* include/votca/ctp/ewaldactor.h
>    <https://github.com/votca/ctp/pull/77/files#diff-14> (4)
>    - *M* include/votca/ctp/fragment.h
>    <https://github.com/votca/ctp/pull/77/files#diff-15> (14)
>    - *M* include/votca/ctp/job.h
>    <https://github.com/votca/ctp/pull/77/files#diff-16> (6)
>    - *M* include/votca/ctp/logger.h
>    <https://github.com/votca/ctp/pull/77/files#diff-17> (4)
>    - *M* include/votca/ctp/molecule.h
>    <https://github.com/votca/ctp/pull/77/files#diff-18> (2)
>    - *M* include/votca/ctp/progressobserver.h
>    <https://github.com/votca/ctp/pull/77/files#diff-19> (10)
>    - *M* include/votca/ctp/qmnblist.h
>    <https://github.com/votca/ctp/pull/77/files#diff-20> (2)
>    - *M* include/votca/ctp/qmpackage.h
>    <https://github.com/votca/ctp/pull/77/files#diff-21> (1)
>    - *M* include/votca/ctp/qmpair.h
>    <https://github.com/votca/ctp/pull/77/files#diff-22> (10)
>    - *M* include/votca/ctp/segment.h
>    <https://github.com/votca/ctp/pull/77/files#diff-23> (10)
>    - *M* include/votca/ctp/segmenttype.h
>    <https://github.com/votca/ctp/pull/77/files#diff-24> (4)
>    - *M* include/votca/ctp/topology.h
>    <https://github.com/votca/ctp/pull/77/files#diff-25> (6)
>    - *M* include/votca/ctp/xinductor.h
>    <https://github.com/votca/ctp/pull/77/files#diff-26> (30)
>    - *M* include/votca/ctp/xinteractor.h
>    <https://github.com/votca/ctp/pull/77/files#diff-27> (8)
>    - *M* include/votca/ctp/xjob.h
>    <https://github.com/votca/ctp/pull/77/files#diff-28> (10)
>    - *M* scripts/CMakeLists.txt
>    <https://github.com/votca/ctp/pull/77/files#diff-29> (4)
>    - *M* scripts/ctp_testsuite.in
>    <https://github.com/votca/ctp/pull/77/files#diff-30> (2)
>    - *M* scripts/ctp_update.in
>    <https://github.com/votca/ctp/pull/77/files#diff-31> (2)
>    - *M* share/doc/CMakeLists.txt
>    <https://github.com/votca/ctp/pull/77/files#diff-32> (10)
>    - *D* share/doc/Doxyfile
>    <https://github.com/votca/ctp/pull/77/files#diff-33> (1513)
>    - *M* src/libctp/CMakeLists.txt
>    <https://github.com/votca/ctp/pull/77/files#diff-34> (7)
>    - *M* src/libctp/calculators/einternal.h
>    <https://github.com/votca/ctp/pull/77/files#diff-35> (2)
>    - *M* src/libctp/calculators/emultipole.h
>    <https://github.com/votca/ctp/pull/77/files#diff-36> (40)
>    - *M* src/libctp/calculators/emultipole_stdal.h
>    <https://github.com/votca/ctp/pull/77/files#diff-37> (38)
>    - *M* src/libctp/calculators/eoutersphere.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-38> (34)
>    - *M* src/libctp/calculators/ianalyze.h
>    <https://github.com/votca/ctp/pull/77/files#diff-39> (2)
>    - *M* src/libctp/calculators/iimport.h
>    <https://github.com/votca/ctp/pull/77/files#diff-40> (32)
>    - *M* src/libctp/calculators/izindo.h
>    <https://github.com/votca/ctp/pull/77/files#diff-41> (20)
>    - *M* src/libctp/calculators/jobwriter.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-42> (2)
>    - *M* src/libctp/calculators/neighborlist.h
>    <https://github.com/votca/ctp/pull/77/files#diff-43> (3)
>    - *M* src/libctp/calculators/pairdump.h
>    <https://github.com/votca/ctp/pull/77/files#diff-44> (2)
>    - *M* src/libctp/calculators/profile.h
>    <https://github.com/votca/ctp/pull/77/files#diff-45> (6)
>    - *M* src/libctp/calculators/qmultipole.h
>    <https://github.com/votca/ctp/pull/77/files#diff-46> (28)
>    - *M* src/libctp/calculators/sandbox.h
>    <https://github.com/votca/ctp/pull/77/files#diff-47> (19)
>    - *M* src/libctp/calculators/stateserver.h
>    <https://github.com/votca/ctp/pull/77/files#diff-48> (6)
>    - *M* src/libctp/calculators/tdump.h
>    <https://github.com/votca/ctp/pull/77/files#diff-49> (2)
>    - *M* src/libctp/calculators/vaverage.h
>    <https://github.com/votca/ctp/pull/77/files#diff-50> (4)
>    - *M* src/libctp/calculators/velocity.h
>    <https://github.com/votca/ctp/pull/77/files#diff-51> (2)
>    - *M* src/libctp/calculators/xmultipole2.h
>    <https://github.com/votca/ctp/pull/77/files#diff-52> (58)
>    - *M* src/libctp/calculators/zmultipole.h
>    <https://github.com/votca/ctp/pull/77/files#diff-53> (26)
>    - *M* src/libctp/ewald3d.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-54> (14)
>    - *M* src/libctp/fragment.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-55> (4)
>    - *M* src/libctp/jobcalculators/dma.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-56> (2)
>    - *M* src/libctp/jobcalculators/ewald.h
>    <https://github.com/votca/ctp/pull/77/files#diff-57> (2)
>    - *M* src/libctp/jobcalculators/qmmm.h
>    <https://github.com/votca/ctp/pull/77/files#diff-58> (6)
>    - *M* src/libctp/jobcalculators/xqmultipole.h
>    <https://github.com/votca/ctp/pull/77/files#diff-59> (6)
>    - *M* src/libctp/molecule.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-60> (4)
>    - *M* src/libctp/orbitals.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-61> (4)
>    - *M* src/libctp/overlap.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-62> (2)
>    - *M* src/libctp/parallelpaircalc.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-63> (8)
>    - *M* src/libctp/parallelxjobcalc.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-64> (14)
>    - *M* src/libctp/poissongrid.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-65> (14)
>    - *M* src/libctp/polarfrag.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-66> (8)
>    - *M* src/libctp/polarseg.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-67> (18)
>    - *M* src/libctp/polarsite.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-68> (12)
>    - *M* src/libctp/progressobserver.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-69> (9)
>    - *M* src/libctp/qmmachine.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-70> (24)
>    - *M* src/libctp/qmnblist.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-71> (4)
>    - *M* src/libctp/qmpackages/turbomole.h
>    <https://github.com/votca/ctp/pull/77/files#diff-72> (6)
>    - *M* src/libctp/qmpair.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-73> (8)
>    - *M* src/libctp/segment.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-74> (2)
>    - *M* src/libctp/statesaversqlite.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-75> (5)
>    - *M* src/libctp/tools/log2mps.h
>    <https://github.com/votca/ctp/pull/77/files#diff-76> (6)
>    - *M* src/libctp/tools/molpol.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-77> (4)
>    - *M* src/libctp/tools/pdb2map.h
>    <https://github.com/votca/ctp/pull/77/files#diff-78> (9)
>    - *M* src/libctp/tools/pdb2top.h
>    <https://github.com/votca/ctp/pull/77/files#diff-79> (5)
>    - *M* src/libctp/tools/ptopreader.h
>    <https://github.com/votca/ctp/pull/77/files#diff-80> (3)
>    - *M* src/libctp/topology.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-81> (8)
>    - *M* src/libctp/xjob.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-82> (10)
>    - *M* src/libctp/xmapper.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-83> (26)
>    - *M* src/tools/Md2QmEngine.cc
>    <https://github.com/votca/ctp/pull/77/files#diff-84> (3)
>
> Patch Links:
>
>    - https://github.com/votca/ctp/pull/77.patch
>    - https://github.com/votca/ctp/pull/77.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/votca/ctp/pull/77>.
>

junghans commented:
I think, I screwed something up in this merge! Can you please push your fix
to make it compile again and I will take care of the remaining  travis
issues!

Am 02.10.2015 1:03 nachm. schrieb "12AngryMen" <notifications@github.com>:
>
> Hallo Christoph,
>
> kannst du mir mal fuer dumme erklaeren was da mit dem merge passiert ist.
> Danke
>
> LG Jens
>
> 2015-10-02 0:59 GMT+02:00 Christoph Junghans <notifications@github.com>:
>
> > Just to avoid that they diverge more.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/votca/ctp/pull/77
> > Commit Summary
> >
> > - Closing old branch
> > - Closing old branch
> > - trying to merge
> > - merged with carl
> > - Fixed 'trimming' to larger number of unoccupied states than available
> > - merge two default heads from Bjoern & Pascal
> > - dummy merge with Denis default branch
> > - dummy merge with Denis even older default branch
> > - cmake: updated modules
> > - cmake: allow version from git, too
> > - remove auto-generated Doxyfile
> > - fixed some format warnings
> > - ctp_testsuite, ctp_update only support python2
> > - fixed many many warnings
> > - gcc warning: added LOG_SAVE
> > - fixed some more warning found by travis
> > - fixed even some more warning found by travis
> > - some more warning found by travis
> > - fixed more warning
> > - fixed a bug introduced in b9ba31880eb5
> > - merge with googlecode
> > - fixed more bugs introduced in b9ba31880eb5
> > - ctp_dump rates, minor bug fix
> > - updated CHANGELOG.md
> > - merge with googlecode
> > - added .travis.yml
> > - hg -> git
> > - cmake: some tweaks for global build
> > - cmake: fixed an out-of-source issue
> > - .travis.yml: update
> > - Merge branch 'master' into wehnerj
> >
> > File Changes
> >
> > - *R* .gitignore <https://github.com/votca/ctp/pull/77/files#diff-0>
> > (5)
> > - *D* .hgtags <https://github.com/votca/ctp/pull/77/files#diff-1> (1)
> > - *A* .travis.yml <https://github.com/votca/ctp/pull/77/files#diff-2>
> > (61)
> > - *R* CHANGELOG.md <https://github.com/votca/ctp/pull/77/files#diff-3>
> > (16)
> > - *M* CMakeLists.txt
> > <https://github.com/votca/ctp/pull/77/files#diff-4> (18)
> > - *D* CMakeModules/FindVOTCA_BOOST.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-5> (37)
> > - *M* CMakeModules/FindVOTCA_CSG.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-6> (22)
> > - *D* CMakeModules/FindVOTCA_KMCOLD.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-7> (59)
> > - *M* CMakeModules/FindVOTCA_MOO.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-8> (6)
> > - *M* CMakeModules/FindVOTCA_TOOLS.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-9> (22)
> > - *M* CMakeModules/hgscript.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-10> (25)
> > - *M* CMakeModules/hgversion.cmake
> > <https://github.com/votca/ctp/pull/77/files#diff-11> (26)
> > - *M* include/votca/ctp/apolarsite.h
> > <https://github.com/votca/ctp/pull/77/files#diff-12> (11)
> > - *M* include/votca/ctp/atom.h
> > <https://github.com/votca/ctp/pull/77/files#diff-13> (37)
> > - *M* include/votca/ctp/ewaldactor.h
> > <https://github.com/votca/ctp/pull/77/files#diff-14> (4)
> > - *M* include/votca/ctp/fragment.h
> > <https://github.com/votca/ctp/pull/77/files#diff-15> (14)
> > - *M* include/votca/ctp/job.h
> > <https://github.com/votca/ctp/pull/77/files#diff-16> (6)
> > - *M* include/votca/ctp/logger.h
> > <https://github.com/votca/ctp/pull/77/files#diff-17> (4)
> > - *M* include/votca/ctp/molecule.h
> > <https://github.com/votca/ctp/pull/77/files#diff-18> (2)
> > - *M* include/votca/ctp/progressobserver.h
> > <https://github.com/votca/ctp/pull/77/files#diff-19> (10)
> > - *M* include/votca/ctp/qmnblist.h
> > <https://github.com/votca/ctp/pull/77/files#diff-20> (2)
> > - *M* include/votca/ctp/qmpackage.h
> > <https://github.com/votca/ctp/pull/77/files#diff-21> (1)
> > - *M* include/votca/ctp/qmpair.h
> > <https://github.com/votca/ctp/pull/77/files#diff-22> (10)
> > - *M* include/votca/ctp/segment.h
> > <https://github.com/votca/ctp/pull/77/files#diff-23> (10)
> > - *M* include/votca/ctp/segmenttype.h
> > <https://github.com/votca/ctp/pull/77/files#diff-24> (4)
> > - *M* include/votca/ctp/topology.h
> > <https://github.com/votca/ctp/pull/77/files#diff-25> (6)
> > - *M* include/votca/ctp/xinductor.h
> > <https://github.com/votca/ctp/pull/77/files#diff-26> (30)
> > - *M* include/votca/ctp/xinteractor.h
> > <https://github.com/votca/ctp/pull/77/files#diff-27> (8)
> > - *M* include/votca/ctp/xjob.h
> > <https://github.com/votca/ctp/pull/77/files#diff-28> (10)
> > - *M* scripts/CMakeLists.txt
> > <https://github.com/votca/ctp/pull/77/files#diff-29> (4)
> > - *M* scripts/ctp_testsuite.in
> > <https://github.com/votca/ctp/pull/77/files#diff-30> (2)
> > - *M* scripts/ctp_update.in
> > <https://github.com/votca/ctp/pull/77/files#diff-31> (2)
> > - *M* share/doc/CMakeLists.txt
> > <https://github.com/votca/ctp/pull/77/files#diff-32> (10)
> > - *D* share/doc/Doxyfile
> > <https://github.com/votca/ctp/pull/77/files#diff-33> (1513)
> > - *M* src/libctp/CMakeLists.txt
> > <https://github.com/votca/ctp/pull/77/files#diff-34> (7)
> > - *M* src/libctp/calculators/einternal.h
> > <https://github.com/votca/ctp/pull/77/files#diff-35> (2)
> > - *M* src/libctp/calculators/emultipole.h
> > <https://github.com/votca/ctp/pull/77/files#diff-36> (40)
> > - *M* src/libctp/calculators/emultipole_stdal.h
> > <https://github.com/votca/ctp/pull/77/files#diff-37> (38)
> > - *M* src/libctp/calculators/eoutersphere.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-38> (34)
> > - *M* src/libctp/calculators/ianalyze.h
> > <https://github.com/votca/ctp/pull/77/files#diff-39> (2)
> > - *M* src/libctp/calculators/iimport.h
> > <https://github.com/votca/ctp/pull/77/files#diff-40> (32)
> > - *M* src/libctp/calculators/izindo.h
> > <https://github.com/votca/ctp/pull/77/files#diff-41> (20)
> > - *M* src/libctp/calculators/jobwriter.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-42> (2)
> > - *M* src/libctp/calculators/neighborlist.h
> > <https://github.com/votca/ctp/pull/77/files#diff-43> (3)
> > - *M* src/libctp/calculators/pairdump.h
> > <https://github.com/votca/ctp/pull/77/files#diff-44> (2)
> > - *M* src/libctp/calculators/profile.h
> > <https://github.com/votca/ctp/pull/77/files#diff-45> (6)
> > - *M* src/libctp/calculators/qmultipole.h
> > <https://github.com/votca/ctp/pull/77/files#diff-46> (28)
> > - *M* src/libctp/calculators/sandbox.h
> > <https://github.com/votca/ctp/pull/77/files#diff-47> (19)
> > - *M* src/libctp/calculators/stateserver.h
> > <https://github.com/votca/ctp/pull/77/files#diff-48> (6)
> > - *M* src/libctp/calculators/tdump.h
> > <https://github.com/votca/ctp/pull/77/files#diff-49> (2)
> > - *M* src/libctp/calculators/vaverage.h
> > <https://github.com/votca/ctp/pull/77/files#diff-50> (4)
> > - *M* src/libctp/calculators/velocity.h
> > <https://github.com/votca/ctp/pull/77/files#diff-51> (2)
> > - *M* src/libctp/calculators/xmultipole2.h
> > <https://github.com/votca/ctp/pull/77/files#diff-52> (58)
> > - *M* src/libctp/calculators/zmultipole.h
> > <https://github.com/votca/ctp/pull/77/files#diff-53> (26)
> > - *M* src/libctp/ewald3d.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-54> (14)
> > - *M* src/libctp/fragment.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-55> (4)
> > - *M* src/libctp/jobcalculators/dma.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-56> (2)
> > - *M* src/libctp/jobcalculators/ewald.h
> > <https://github.com/votca/ctp/pull/77/files#diff-57> (2)
> > - *M* src/libctp/jobcalculators/qmmm.h
> > <https://github.com/votca/ctp/pull/77/files#diff-58> (6)
> > - *M* src/libctp/jobcalculators/xqmultipole.h
> > <https://github.com/votca/ctp/pull/77/files#diff-59> (6)
> > - *M* src/libctp/molecule.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-60> (4)
> > - *M* src/libctp/orbitals.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-61> (4)
> > - *M* src/libctp/overlap.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-62> (2)
> > - *M* src/libctp/parallelpaircalc.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-63> (8)
> > - *M* src/libctp/parallelxjobcalc.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-64> (14)
> > - *M* src/libctp/poissongrid.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-65> (14)
> > - *M* src/libctp/polarfrag.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-66> (8)
> > - *M* src/libctp/polarseg.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-67> (18)
> > - *M* src/libctp/polarsite.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-68> (12)
> > - *M* src/libctp/progressobserver.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-69> (9)
> > - *M* src/libctp/qmmachine.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-70> (24)
> > - *M* src/libctp/qmnblist.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-71> (4)
> > - *M* src/libctp/qmpackages/turbomole.h
> > <https://github.com/votca/ctp/pull/77/files#diff-72> (6)
> > - *M* src/libctp/qmpair.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-73> (8)
> > - *M* src/libctp/segment.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-74> (2)
> > - *M* src/libctp/statesaversqlite.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-75> (5)
> > - *M* src/libctp/tools/log2mps.h
> > <https://github.com/votca/ctp/pull/77/files#diff-76> (6)
> > - *M* src/libctp/tools/molpol.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-77> (4)
> > - *M* src/libctp/tools/pdb2map.h
> > <https://github.com/votca/ctp/pull/77/files#diff-78> (9)
> > - *M* src/libctp/tools/pdb2top.h
> > <https://github.com/votca/ctp/pull/77/files#diff-79> (5)
> > - *M* src/libctp/tools/ptopreader.h
> > <https://github.com/votca/ctp/pull/77/files#diff-80> (3)
> > - *M* src/libctp/topology.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-81> (8)
> > - *M* src/libctp/xjob.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-82> (10)
> > - *M* src/libctp/xmapper.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-83> (26)
> > - *M* src/tools/Md2QmEngine.cc
> > <https://github.com/votca/ctp/pull/77/files#diff-84> (3)
> >
> > Patch Links:
> >
> > - https://github.com/votca/ctp/pull/77.patch
> > - https://github.com/votca/ctp/pull/77.diff
>
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/votca/ctp/pull/77>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub.

12AngryMen commented:
hey ist gemacht.

ich werde keine pull request mehr akzeptieren. :) Danke

2015-10-02 15:26 GMT+02:00 Christoph Junghans <notifications@github.com>:

> I think, I screwed something up in this merge! Can you please push your fix
> to make it compile again and I will take care of the remaining travis
> issues!
>
> Am 02.10.2015 1:03 nachm. schrieb "12AngryMen" <notifications@github.com>:
> >
> > Hallo Christoph,
> >
> > kannst du mir mal fuer dumme erklaeren was da mit dem merge passiert ist.
> > Danke
> >
> > LG Jens
> >
> > 2015-10-02 0:59 GMT+02:00 Christoph Junghans <notifications@github.com>:
> >
> > > Just to avoid that they diverge more.
> > > ------------------------------
> > > You can view, comment on, or merge this pull request online at:
> > >
> > > https://github.com/votca/ctp/pull/77
> > > Commit Summary
> > >
> > > - Closing old branch
> > > - Closing old branch
> > > - trying to merge
> > > - merged with carl
> > > - Fixed 'trimming' to larger number of unoccupied states than available
> > > - merge two default heads from Bjoern & Pascal
> > > - dummy merge with Denis default branch
> > > - dummy merge with Denis even older default branch
> > > - cmake: updated modules
> > > - cmake: allow version from git, too
> > > - remove auto-generated Doxyfile
> > > - fixed some format warnings
> > > - ctp_testsuite, ctp_update only support python2
> > > - fixed many many warnings
> > > - gcc warning: added LOG_SAVE
> > > - fixed some more warning found by travis
> > > - fixed even some more warning found by travis
> > > - some more warning found by travis
> > > - fixed more warning
> > > - fixed a bug introduced in b9ba31880eb5
> > > - merge with googlecode
> > > - fixed more bugs introduced in b9ba31880eb5
> > > - ctp_dump rates, minor bug fix
> > > - updated CHANGELOG.md
> > > - merge with googlecode
> > > - added .travis.yml
> > > - hg -> git
> > > - cmake: some tweaks for global build
> > > - cmake: fixed an out-of-source issue
> > > - .travis.yml: update
> > > - Merge branch 'master' into wehnerj
> > >
> > > File Changes
> > >
> > > - *R* .gitignore <https://github.com/votca/ctp/pull/77/files#diff-0>
> > > (5)
> > > - *D* .hgtags <https://github.com/votca/ctp/pull/77/files#diff-1> (1)
> > > - *A* .travis.yml <https://github.com/votca/ctp/pull/77/files#diff-2>
> > > (61)
> > > - *R* CHANGELOG.md <https://github.com/votca/ctp/pull/77/files#diff-3>
> > > (16)
> > > - *M* CMakeLists.txt
> > > <https://github.com/votca/ctp/pull/77/files#diff-4> (18)
> > > - *D* CMakeModules/FindVOTCA_BOOST.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-5> (37)
> > > - *M* CMakeModules/FindVOTCA_CSG.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-6> (22)
> > > - *D* CMakeModules/FindVOTCA_KMCOLD.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-7> (59)
> > > - *M* CMakeModules/FindVOTCA_MOO.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-8> (6)
> > > - *M* CMakeModules/FindVOTCA_TOOLS.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-9> (22)
> > > - *M* CMakeModules/hgscript.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-10> (25)
> > > - *M* CMakeModules/hgversion.cmake
> > > <https://github.com/votca/ctp/pull/77/files#diff-11> (26)
> > > - *M* include/votca/ctp/apolarsite.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-12> (11)
> > > - *M* include/votca/ctp/atom.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-13> (37)
> > > - *M* include/votca/ctp/ewaldactor.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-14> (4)
> > > - *M* include/votca/ctp/fragment.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-15> (14)
> > > - *M* include/votca/ctp/job.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-16> (6)
> > > - *M* include/votca/ctp/logger.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-17> (4)
> > > - *M* include/votca/ctp/molecule.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-18> (2)
> > > - *M* include/votca/ctp/progressobserver.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-19> (10)
> > > - *M* include/votca/ctp/qmnblist.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-20> (2)
> > > - *M* include/votca/ctp/qmpackage.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-21> (1)
> > > - *M* include/votca/ctp/qmpair.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-22> (10)
> > > - *M* include/votca/ctp/segment.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-23> (10)
> > > - *M* include/votca/ctp/segmenttype.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-24> (4)
> > > - *M* include/votca/ctp/topology.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-25> (6)
> > > - *M* include/votca/ctp/xinductor.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-26> (30)
> > > - *M* include/votca/ctp/xinteractor.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-27> (8)
> > > - *M* include/votca/ctp/xjob.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-28> (10)
> > > - *M* scripts/CMakeLists.txt
> > > <https://github.com/votca/ctp/pull/77/files#diff-29> (4)
> > > - *M* scripts/ctp_testsuite.in
> > > <https://github.com/votca/ctp/pull/77/files#diff-30> (2)
> > > - *M* scripts/ctp_update.in
> > > <https://github.com/votca/ctp/pull/77/files#diff-31> (2)
> > > - *M* share/doc/CMakeLists.txt
> > > <https://github.com/votca/ctp/pull/77/files#diff-32> (10)
> > > - *D* share/doc/Doxyfile
> > > <https://github.com/votca/ctp/pull/77/files#diff-33> (1513)
> > > - *M* src/libctp/CMakeLists.txt
> > > <https://github.com/votca/ctp/pull/77/files#diff-34> (7)
> > > - *M* src/libctp/calculators/einternal.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-35> (2)
> > > - *M* src/libctp/calculators/emultipole.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-36> (40)
> > > - *M* src/libctp/calculators/emultipole_stdal.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-37> (38)
> > > - *M* src/libctp/calculators/eoutersphere.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-38> (34)
> > > - *M* src/libctp/calculators/ianalyze.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-39> (2)
> > > - *M* src/libctp/calculators/iimport.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-40> (32)
> > > - *M* src/libctp/calculators/izindo.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-41> (20)
> > > - *M* src/libctp/calculators/jobwriter.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-42> (2)
> > > - *M* src/libctp/calculators/neighborlist.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-43> (3)
> > > - *M* src/libctp/calculators/pairdump.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-44> (2)
> > > - *M* src/libctp/calculators/profile.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-45> (6)
> > > - *M* src/libctp/calculators/qmultipole.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-46> (28)
> > > - *M* src/libctp/calculators/sandbox.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-47> (19)
> > > - *M* src/libctp/calculators/stateserver.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-48> (6)
> > > - *M* src/libctp/calculators/tdump.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-49> (2)
> > > - *M* src/libctp/calculators/vaverage.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-50> (4)
> > > - *M* src/libctp/calculators/velocity.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-51> (2)
> > > - *M* src/libctp/calculators/xmultipole2.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-52> (58)
> > > - *M* src/libctp/calculators/zmultipole.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-53> (26)
> > > - *M* src/libctp/ewald3d.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-54> (14)
> > > - *M* src/libctp/fragment.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-55> (4)
> > > - *M* src/libctp/jobcalculators/dma.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-56> (2)
> > > - *M* src/libctp/jobcalculators/ewald.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-57> (2)
> > > - *M* src/libctp/jobcalculators/qmmm.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-58> (6)
> > > - *M* src/libctp/jobcalculators/xqmultipole.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-59> (6)
> > > - *M* src/libctp/molecule.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-60> (4)
> > > - *M* src/libctp/orbitals.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-61> (4)
> > > - *M* src/libctp/overlap.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-62> (2)
> > > - *M* src/libctp/parallelpaircalc.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-63> (8)
> > > - *M* src/libctp/parallelxjobcalc.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-64> (14)
> > > - *M* src/libctp/poissongrid.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-65> (14)
> > > - *M* src/libctp/polarfrag.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-66> (8)
> > > - *M* src/libctp/polarseg.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-67> (18)
> > > - *M* src/libctp/polarsite.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-68> (12)
> > > - *M* src/libctp/progressobserver.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-69> (9)
> > > - *M* src/libctp/qmmachine.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-70> (24)
> > > - *M* src/libctp/qmnblist.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-71> (4)
> > > - *M* src/libctp/qmpackages/turbomole.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-72> (6)
> > > - *M* src/libctp/qmpair.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-73> (8)
> > > - *M* src/libctp/segment.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-74> (2)
> > > - *M* src/libctp/statesaversqlite.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-75> (5)
> > > - *M* src/libctp/tools/log2mps.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-76> (6)
> > > - *M* src/libctp/tools/molpol.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-77> (4)
> > > - *M* src/libctp/tools/pdb2map.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-78> (9)
> > > - *M* src/libctp/tools/pdb2top.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-79> (5)
> > > - *M* src/libctp/tools/ptopreader.h
> > > <https://github.com/votca/ctp/pull/77/files#diff-80> (3)
> > > - *M* src/libctp/topology.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-81> (8)
> > > - *M* src/libctp/xjob.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-82> (10)
> > > - *M* src/libctp/xmapper.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-83> (26)
> > > - *M* src/tools/Md2QmEngine.cc
> > > <https://github.com/votca/ctp/pull/77/files#diff-84> (3)
> > >
> > > Patch Links:
> > >
> > > - https://github.com/votca/ctp/pull/77.patch
> > > - https://github.com/votca/ctp/pull/77.diff
> >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/votca/ctp/pull/77>.
>
> >
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/votca/ctp/pull/77#issuecomment-145019717>.
>
